### ISSUE TITLE
Design Save-the-dates

### ISSUE THREAD

**Issue by CarolynT2017** [azoff/wedding#38]
Use Save-the-dates as a way to set the theme for our wedding.

**Comment by azoff** [comment]
Things we know for sure:

1) It will have the date of the wedding on it
2) It will have the location of the wedding on it
3) It will have our faces on it 
4) It will have a special link that can be made unique to each recipient.

The link will send them to a special website where they will see:

1) A photo of us and them
2) A text field with their name(s), this will match the eventual invitation
3) A checkbox to make sure that their mailing address is correct
4) A dropdown to select communication preferences (phone, email, nope)

What this all means is that we're going to need:

- [ ] A list of all guest's addresses (to be added to guest list)
- [ ] A photo of us for the save-the-date
- [ ] A design for the save-the-date
- [ ] A spreadsheet to power the website
- [ ] The landing page site itself

**Comment by azoff** [comment]
I've started the save the date designs. I'm going to put together a vertical and horizontal version, so that you have some options. I've made them a standard (A6) size, so that we can have them printed at a local printer/embosser and then buy some nice C6 sized envelopes to send them in.

As for providing the links, I think we can just make some slightly smaller cards out of some Craft paper with a place to put their links. I'll have those for review soon
